### ISSUE TITLE
test failure when scipy installed without matplotlib in Python 2

### ISSUE THREAD

**Issue by afif-elghraoui** [opencobra/cobrapy#224]
When running unit tests at build time with scipy installed without matplotlib, I get the following error:
```
======================================================================
ERROR: test_phenotype_phase_plane (cobra.test.flux_analysis.TestCobraFluxAnalysis)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/<<PKGBUILDDIR>>/cobra/test/flux_analysis.py", line 250, in test_phenotype_phase_plane
    self.skip("can't test plots without matplotlib")
AttributeError: 'TestCobraFluxAnalysis' object has no attribute 'skip'

----------------------------------------------------------------------
Ran 122 tests in 33.498s

FAILED (errors=1, skipped=9)
```
I got around this in building the Debian package by applying the following patch:

```diff
--- python-cobra.orig/cobra/test/flux_analysis.py
+++ python-cobra/cobra/test/flux_analysis.py
@@ -247,7 +247,7 @@
         self.assertAlmostEqual(data.growth_rates.max(), 1.20898, places=4)
         self.assertAlmostEqual(abs(data.growth_rates[0, :]).max(), 0, places=4)
         if matplotlib is None:
-            self.skip("can't test plots without matplotlib")
+            return
         data.plot()
```

**Comment by aebrahim** [comment]
Yup sorry about that. Fixed in 2ddffbe26b0f9b4050c4eedf989f99f07fe9def4

**Comment by afif-elghraoui** [comment]
Oh, haha...the commit right after the release. I'll make sure to check those next time. This release was so recent that I hadn't expected it to be fixed already. Thanks
